articles/azure-functions/functions-networking-options.md

```diff
@@ -125,8 +125,8 @@ Considerations for the [Elastic Premium](./functions-premium-plan.md), [Dedicate
 * The feature is available for Elastic Premium and App Service Premium V2 and Premium V3. It's also available in Standard but only from newer App Service deployments. If you are on an older deployment, you can only use the feature from a Premium V2 App Service plan. If you want to make sure you can use the feature in a Standard App Service plan, create your app in a Premium V3 App Service plan. Those plans are only supported on our newest deployments. You can scale down if you desire after that.
 * The feature can't be used by Isolated plan apps that are in an App Service Environment.
 * The app and the virtual network must be in the same region.
-* The feature requires an unused subnet that's a /28 or larger in an Azure Resource Manager virtual network.
-* The integration subnet can be used by only one App Service plan.
+* The feature requires a subnet that's a /28 or larger in an Azure Resource Manager virtual network.
```

Comment:
@rapopescu16 @im-samz - if the subnet is being used by other services (like a private endpoint into, say, service bus or storage) then the virtual network integration would fail. While I understand that the subnet can be shared between multiple app service plans now, it can't also be delegated to other services right? I think this distinction would still be worth calling out. For [Flex Consumption in the docs](https://learn.microsoft.com/en-us/azure/azure-functions/flex-consumption-how-to#enable-virtual-network-integration) we made this distinction by using two different bullet points:
![image](https://github.com/user-attachments/assets/8cdc4e57-8f0a-402e-861b-470a4267baee)

Reply:
I addressed this in a separate PR wtih Glenn.

Reply:
@im-samz Should we close this PR if it was addressed in the private repo?